parm/archive/master_gdas.yaml.j2

```diff
@@ -97,3 +97,10 @@ datasets:
 
     # End of restart checking
 {% endif %}
+
+# Archive the EXPDIR if requested
+{% if archive_expdir %}
+{% filter indent(width=4) %}
+{% include "expdir.yaml.j2" %}
+{% endfilter %}
+{% endif %}
```

Comment:
Should only need this in one of gdas or gfs, but there is a bit of a coordination problem since either can be run without the other.